lib/shift/circuit_breaker/circuit_handler.rb

```diff
@@ -109,6 +109,9 @@ def handle_exception(exception, fallback)
         record_error
         set_state
         log_errors(exception)
+        # Increments exceptions; useful for alerting
+        monitor.record_exception(name)
+        # Increments total calls with current state e.g. open/closed
         monitor.record_metric(name, state)
```

Comment:
I don't think it's worthwhile recording 2 metrics for both errored and opened, the open state should remain following errors for a period of time.

There's two ways you could acheive this, using your new method:

```suggestion
        # Increments exceptions; useful for alerting
        monitor.record_exception(name)
```

Or just reusing the existing method:

```suggestion
        # Increments exceptions; useful for alerting
        monitor.record_metric(name, "errored")
```